lib/controller/matlab/wb_supervisor_field_disable_sf_tracking.m

```diff
@@ -0,0 +1,6 @@
+function result = wb_supervisor_field_disable_sf_tracking(field)
+% Usage: wb_supervisor_field_disable_sf_tracking(field)
+% Matlab API for Webots
+% Online documentation is available <a href="https://www.cyberbotics.com/doc/reference/supervisor">here</a>
+
+result = calllib('libController', 'wb_supervisor_field_disable_sf_tracking', field);
```

Comment:
Since these matlab API functions should be automatically generated by the new `mgenerate.py` script, they should not be committed here, but ignored (`.gitignore`). See https://github.com/cyberbotics/webots/pull/3114. I believe the script has an option to update the local `.gitignore` file.

Reply:
Maybe we should exclude all files in the folder except the ones which need custom implementation:
```
*.m
!wb_camera_get_image.m
```

Reply:
@BenjaminDeleze: do you think it would be better to do so?

Reply:
It would definitely reduce the size of the .gitignore (from ~500 lines to ~100) but I am not sure that it is possible to automate the maintenance of such a .gitignore